crates/runtime/Cargo.toml

```diff
@@ -50,6 +50,7 @@ features = [
 
 [dev-dependencies]
 once_cell = { workspace = true }
+proptest = "1.2.0"
```

Comment:
Guess I should not upgrade this package. It results in a huge vet certification and the addition of `unarray` which has unsafe code to audit. `wiggle` uses version 1.0.0 which is declared exempt as `safe-to-deploy`. I think we should only have to audit this as `safe-to-run` since it is a test dependency but once declared `safe-to-deploy` it would seem that we need to vet at the higher level.